src/client/renderingengine.cpp

```diff
@@ -410,6 +410,27 @@ void RenderingEngine::_draw_load_screen(const std::wstring &text,
 	guitext->remove();
 }
 
+/*
+	Draws a screen with clouds.
+	This is basically just to make the GUI drawn.
+*/
+void RenderingEngine::_draw_screen_with_clouds(gui::IGUIEnvironment *guienv,
+		float dtime, bool clouds)
```

Comment:
suggested rename:
<pre>
/*
 	Draws the menu scene including (optional) cloud background
 */
 void RenderingEngine::_draw_menu_scene(gui::IGUIEnvironment *guienv,
 		float dtime, bool clouds)
</pre>